tests/test_unclassified_job.py

```diff
@@ -12,7 +12,7 @@
 def test_unclassified_failure(base_url, selenium):
     """Open resultset page and search for next unclassified failure"""
     page = TreeherderPage(selenium, base_url).open()
-    assert page.unclassified_failure_count > 0
+    page.wait_for_unclassified_failures_to_load()
```

Comment:
I wonder if we even need this assert/wait. It seems these tests are more concerned with opening an unclassified failure, and that seems to work even before the count is finished loading.